flake.nix

```diff
@@ -5,9 +5,10 @@
     nixpkgs.follows = "bot-plutus-interface/nixpkgs";
     haskell-nix.follows = "bot-plutus-interface/haskell-nix";
 
-    bot-plutus-interface.url = "github:mlabs-haskell/bot-plutus-interface?ref=gergely/vasil";
-
     cardano-node.url = "github:input-output-hk/cardano-node?ref=1.35.3";
+
+    bot-plutus-interface.url = "github:mlabs-haskell/bot-plutus-interface";
+    bot-plutus-interface.inputs.cardano-node.follows = "cardano-node";
```

Comment:
I guess the `cardano-node` input in the flake here should just follow `bot-plutus-interface.inputs.cardano-node` - I reckon  it'd be the same in this case? Can we trust BPI to always follow the right node input?

Reply:
rn it's the same, but I don't know if we can always trust BPI